.changeset/full-showers-kiss.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-components": major
+---
+
+- Removed `sass` and `ember-cli-sass` as dependencies
+- Added `sass` as a dev-dependency
```

Comment:
We generally don't surface our `devDependencies` to consumers. On all major changes we tend to offer brief but useful guidance on how to overcome the potential impact of these changes.
For example:

```md
Removed `sass` and `ember-cli-sass` dependencies. Consumers using `sass` in their projects should make sure it's added as a direct dependency to their project.
```